plugins/inputs/supervisor/README.md

```diff
@@ -0,0 +1,87 @@
+# Supervisor Input Plugin
+
+This plugin gather information about processes that running under supervisor using XML-RPC API
+
+Plugin minimum tested version: 1.17.0
```

Comment:
I don't see this version in their [release history](http://supervisord.org/#release-history)?

Reply:
I think i accidently wrote here minimum version of telegraf itself, minimum supervisor version i actually tested plugin with was 3.3.5 (from debian buster repositories). Will be fixed in near time.